client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -10,26 +10,40 @@ import MediaBlock from '../blocks/MediaBlock';
  */
 const EmbedBlock = props => {
   const { entity, onRemoveEntity } = props.blockProps;
-  const { url, title, thumbnail } = entity.getData();
+  const { url, title, thumbnail, providerName, authorName } = entity.getData();
+
+  if (thumbnail) {
+    return (
+      <MediaBlock {...props} src={thumbnail} alt="">
+        {url ? (
+          <a
+            className="Tooltip__link EmbedBlock__link"
+            href={url}
+            title={url}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {title}
+          </a>
+        ) : null}
+
+        <button className="button button-secondary no Tooltip__button" onClick={onRemoveEntity}>
+          {STRINGS.DELETE}
+        </button>
+      </MediaBlock>
+    );
+  }
 
   return (
-    <MediaBlock {...props} src={thumbnail} alt="">
-      {url ? (
-        <a
-          className="Tooltip__link EmbedBlock__link"
-          href={url}
-          title={url}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {title}
-        </a>
-      ) : null}
-
-      <button className="button button-secondary no Tooltip__button" onClick={onRemoveEntity}>
+    <strong>
+      <a href={url} target={'_blank'}>
+        {providerName + ' - ' + authorName + '  '}
+      </a>
+
+      <button className="button button-secondary no Tooltip__button" onClick={onRemoveEntity} >
```

Comment:
I think this should stay within the tooltip, so the block’s UI isn’t distracting while entering content.